README.md

```diff
@@ -30,26 +22,67 @@ defp deps do
 end
 ```
 
-### Setup with Plug or Phoenix
+### Setup with Plug and Phoenix
+Capturing errors in Plug applications is done with `Sentry.PlugContext` and `Sentry.PlugCapture`. `Sentry.PlugContext` adds contextual metadata from the current request which is then included in errors that are captured and reported by `Sentry.PlugCapture`.
 
-In your Plug.Router or Phoenix.Router, add the following lines:
+If you are using Phoenix, first add `Sentry.PlugCapture` above the `use Phoenix.Endpoint` line in your endpoint file. `Sentry.PlugContext` should be added below `Plug.Parsers`.
```

Comment:
IIRC, we had to add it above if we were going to render something, but we are not going to render anything, so the order most likely does not matter. :)

Reply:
I thought it was necessary because PlugContext may try to manipulate/scrub the body of the request, and if it wasn't parsed before then it could cause issues?

Reply:
Sorry, I was not clear. :D I wanted to say `Sentry.PlugCapture` does not need to be above/before "use Phoenix.Endpoint"/"use Plug.Router" anymore. :) But the parsers one is recommended, yes. :)

Reply:
ohh, my fault, thank you, yeah

Reply:
I've written a few more tests, and it seems like PlugCapture does have to go first in the Endpoint. If you flip the lines [here](https://github.com/getsentry/sentry-elixir/pull/402/files#diff-24ca02f3e0bdcfc2284daa120a1c5889R19-R20) and run the tests it fails because the ErrorView doesn't get anything from last_event_id.

Reply:
Ah, good call!